mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/modules/CassandraAttachmentModule.java

```diff
@@ -48,7 +50,14 @@ public CassandraAttachmentModule() {
                     .addColumn(CassandraAttachmentTable.TYPE, text())
                     .addColumn(CassandraAttachmentTable.SIZE, bigint())
                     .withOptions()
-                    .comment("Holds attachment for fast attachment retrieval")));
+                    .comment("Holds attachment for fast attachment retrieval")),
+            new CassandraTable(CassandraAttachmentMessageIdTable.TABLE_NAME,
+                SchemaBuilder.createTable(CassandraAttachmentMessageIdTable.TABLE_NAME)
+                    .ifNotExists()
+                    .addPartitionKey(CassandraAttachmentMessageIdTable.ATTACHMENT_ID, text())
```

Comment:
See https://github.com/linagora/james-project/pull/960/commits/762be3bb153a44e9bb46e696c89e435de1b2ad8e before @mbaechler  complains....

We can introduce another column holding the UUID version of the AttachmentId and use it as Primary key (more efficient).

Reply:
done